python/ml4ir/base/tests/test_auto_dag_network.py

```diff
@@ -0,0 +1,117 @@
+import unittest
+
+from tensorflow.keras.layers import Layer
+
+from ml4ir.base.model.architectures.auto_dag_network import LayerNode, get_layer_subclasses, CycleFoundException, \
+    LayerGraph
+
+
+class UserDefinedTestLayerGlobal(Layer):
+    pass
+
+
+class TestGetLayerSubclasses(unittest.TestCase):
```

Comment:
@arvindsrikantan Can also add a test for a keras native layer to be "complete"

Reply:
@lastmansleeping , didn't quite get you. What do you mean by complete? Have all methods defined?

Reply:
Added keras native layers